gradle/instrumentation.gradle

```diff
@@ -37,17 +36,20 @@ shadowJar {
     relocatePackages(it)
 }
 
+evaluationDependsOn(":testing:agent-for-testing")
+
 tasks.withType(Test).configureEach {
+    dependsOn shadowJar
+    dependsOn ":testing:agent-for-testing:shadowJar"
+
     jvmArgs "-Dotel.javaagent.debug=true"
-    jvmArgs "-javaagent:${configurations.testAgent.files.first().absolutePath}"
+    jvmArgs "-javaagent:${project(":testing:agent-for-testing").tasks.shadowJar.archiveFile.get().asFile.absolutePath}"
```

Comment:
I think you should change it back :) https://docs.gradle.org/current/userguide/cross_project_publications.html

Reply:
I think I fixed it 😅